osgar/tools/lidarview.py

```diff
@@ -11,6 +11,7 @@
 
 from osgar.logger import LogReader, lookup_stream_id, lookup_stream_names
 from osgar.lib.serialize import deserialize
+from osgar.lib.config import get_class_by_name
```

Comment:
We might try to come up with a better name since it can get anything, not just class.

Reply:
OK, if you have any suggestion let me know ... thanks.